Classes/Service/VarnishBanService.php

```diff
@@ -66,10 +61,14 @@ public function initializeObject()
         array_walk($varnishUrls, function (&$varnishUrl) {
             $varnishUrl = rtrim($varnishUrl, '/');
         });
-        $this->varnishProxyClient = new ProxyClient\Varnish($varnishUrls);
-        $this->varnishProxyClient->setDefaultBanHeader('X-Site', $this->tokenStorage->getToken());
+        $httpDispatcher = new FOS\HttpCache\ProxyClient\HttpDispatcher($varnishUrls, '');
```

Comment:
```suggestion
        $httpDispatcher = new ProxyClient\HttpDispatcher($varnishUrls);
```
The default value for the baseUri is an empty string already